core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -297,6 +308,7 @@ static function (array $value): array {
                     ->end()
                 ->end()
                 ->variableNode('localconfig')
+                    ->info('Allows to set TL_CONFIG variables. Note that any configuration set here can not be edited in the back end anymore.')
```

Comment:
Suggestion: *Allows to set TL_CONFIG variables. Note that any property set here will override the `localconfig.php` file, so changing these in the Contao back end will not have any effect.*